js/adapt-contrib-ios-frameset.js

```diff
@@ -4,12 +4,17 @@
 */
 $(function() {
     // Detect if we're in a FRAME or IFRAME
-    if (window.frameElement && (window.frameElement.nodeName == "FRAME" || window.frameElement.nodeName == "IFRAME")) {
+    if (window.self !== window.top) {
+        if (window.top && !$('html').hasClass('ie')) {
+            try {
+                if (!window.top.document.querySelector("meta[name=viewport]")) {
+                    $("head", window.top.document).append('<meta name="viewport" content="initial-scale=1.0, user-scalable=yes">');
+                } else {
+                    $("meta[name='viewport']", window.top.document).attr("content", "initial-scale=1.0, user-scalable=yes");
+                }
+            } catch (e) {
 
```

Comment:
Is there any value in logging a caught exception here?

Reply:
No, it's fine.  We're not going to do anything with it and it's a circumstance we're trying to handle (by ignoring).